lib/HttpSignatureStrategy.js

```diff
@@ -229,15 +229,24 @@ function getHttpsPublicKey(publicKey, callback) {
   async.auto({
     dereferencePublicKey: function(callback) {
       jsonRequest.get(publicKey, function(err, res) {
-        callback(err, res.body);
+        if(err) {
+          return callback(new BedrockError(
+            'PublicKey URL unavailable.',
```

Comment:
While we're making these changes I think we should change the text from `'PublicKey ...` to `'Public key ...`.